public/js/navbar.js

```diff
@@ -1,13 +1,28 @@
+
 var getPoints = document.querySelector(".getPoints");
+var xpBar = document.querySelector("#xp-bar");
+
+// code for the dropdown menu
+(() => {
+    const menuBtn = document.querySelector('.menu-button');
+    const popoutMenu = document.querySelector('.popout-menu');
+    menuBtn.addEventListener('click', () =>
+        toggleMenu(popoutMenu, menuBtn)
+    );
+})();
```

Comment:
I would say to use `function` over `=>` here for [browser compatibility](https://caniuse.com/#search=arrow)